src/components/atoms/Link/Link.tsx

```diff
@@ -3,10 +3,7 @@ import * as React from "react";
 import styled, { withTheme } from "styled-components";
 import { getFontFromTheme } from "../../../theme/provider";
 
-export interface LinkProps
-  extends React.ButtonHTMLAttributes<HTMLAnchorElement> {
-  children?: React.ReactNode;
-}
+export type LinkProps = React.AnchorHTMLAttributes<HTMLAnchorElement>;
```

Comment:
in the previous version, I used href for the `Link` component but I converted it to `onClick` handler, I can revert these changes but I think we can leave it because this line solve the problem that exists